reputation/src/lib.rs

```diff
@@ -5,10 +5,12 @@
 #![deny(non_camel_case_types)]
 #![deny(non_snake_case)]
 #![deny(unused_mut)]
-#![deny(missing_docs)]
 
 pub mod trs;
 pub use trs::TotalReputationSet;
 
 pub mod ars;
 pub use ars::ActiveReputationSet;
+
+/// Module containing error definitions
+pub mod error;
```

Comment:
Since you are creating a error module, could you move the errors from `trs.rs` there? (`NonSortedAlpha` and `RepError`)